recipe/meta.yaml

```diff
@@ -29,20 +28,17 @@ requirements:
     - python
     - pip
     # host-dependency only needed for compiling CUDA extensions
-    - pytorch {{ torch_version }}.*
+    - pytorch                         # [cuda_compiler_version != "None"]
```

Comment:
@hmaarfk, I would have expected the rerender to pick up a pytorch-pin for the CUDA-builds now that conda-forge/conda-forge-pinning-feedstock#1890 has been merged. Do you know why that's not happening?

I also tried a selector-less version, to no avail:
```diff
-    - pytorch                         # [cuda_compiler_version != "None"]
+    {% if cuda_compiler_version != "None" %}
+    # don't use selector so this can pick up the global pinning
+    - pytorch
+    {% endif %}
```

Reply:
I would just leave `pytorch` in the host section even for CPU builds.

For GPU/CPU, because upstream decided to move away from the `-gpu` suffix, I've started doing the following
https://github.com/conda-forge/torchvision-feedstock/blob/main/recipe/meta.yaml#L55

You'll also want to pull in the migrator.

Reply:
Thanks for the response!

> You'll also want to pull in the migrator.

Duh 🤦 

> I would just leave `pytorch` in the host section even for CPU builds.

What's your thinking behind that - it's genuinely not necessary for the CPU builds. Why complicate the dependencies? Or is it just that you'd prefer a consistent run-export across CPU/GPU?

> For GPU/CPU, because upstream decided to move away from the `-gpu` suffix, I've started doing the following

Fine by me.

Reply:
Run export handles the dependencies, and likely helps the migrator do it's job more readily.